drivers/char/mem.c

```diff
@@ -797,8 +797,14 @@ static const struct memdev {
 #endif
 	 [5] = { "zero", 0666, &zero_fops, 0 },
 	 [7] = { "full", 0666, &full_fops, 0 },
+#if CONFIG_BSD_RANDOM
 	 [8] = { "random", 0666, &random_fops, 0 },
 	 [9] = { "urandom", 0666, &urandom_fops, 0 },
+#else
+	 [8] = { "bsd_urandom", 0666, &bsd_urandom_fops, 0 },
+	 [9] = { "bsd_urandom", 0666, &bsd_urandom_fops, 0 },
+#endif
```

Comment:
If I understand this correctly, this file doesn't create the actual `/dev/urandom` and `/dev/random`, it just internally registers the internal things so when you `mknod 1 8 /dev/random` and `mknod 1 9 /dev/urandom` it knows where to find the `file_operations` corresponding to them. Will change the the names to still be random/urandom though.